### DISCUSSION TITLE
Ambiguity in Data Context in Basics

### DISCUSSION THREAD

**Discussion by SamHobbsOrg** [AvaloniaUI/Avalonia#13573] [Q&A]
In <a href="https://docs.avaloniaui.net/docs/next/basics/data/data-binding/data-context">Data Context | Avalonia UI</a> is _Avalonia performs a hierarchical search_ but it does not say whether it is an upward or downward or all-around search. I assume it is an upward search but if not then it is especially important for the documentation to be clarified.

Sorry, I should have posted this as a documentation issue, correct?

**Comment by timunie** [comment]
Discussion is way more relaxed than issues. So I'm fine with it. 

The direction is upward 👆 as the image should suggest. However if you find it can be more clarified, feel free ro provide a PR to fix the docs 😉
